include/DoViProcessor.h

```diff
@@ -6,7 +6,10 @@
 
 #include "rpu_parser.h"
 #include <vector>
+#ifdef _WIN32
```

Comment:
#ifndef NOMINMAX
#define NOMINMAX
#endif
#ifndef WIN32_LEAN_AND_MEAN
#define WIN32_LEAN_AND_MEAN
#endif

This way min/max are excluded from windows.h.